makefile

```diff
@@ -0,0 +1,33 @@
+include .env
+
+.PHONY: download build run stop venvs start_dagster test
```

Comment:
This is also not in its final form but this is more or less what I am using now to run everything. Later I will look into transferring this to 'just'